### DISCUSSION TITLE
请问这个item怎么关闭

### DISCUSSION THREAD

**Discussion by abcvav** [aidenlx/obsidian-zotero#276] [Help]
不知道怎么打开了，右键点击也没有关闭按钮，不知道怎么才能关上
![image](https://github.com/PKM-er/obsidian-zotlit/assets/47599849/e7d70860-6ebe-4e1a-9d82-27e4edead101)

**Comment by aidenlx** [comment]
这是 v1 的 item details 视图，v2 里已经不再提供了。

v2 的侧边栏视图（标注视图、模板数据浏览器）都是普通的 Obsidian 标签页，点标签页上的关闭按钮即可关掉，也可以从命令面板按需打开。

这条属于 v1 的界面，关掉了。

v2 beta 现已发布：https://zotlit.aidenlx.site/docs/install-zotlit
迁移指南：https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
